### PR TITLE
feat: add quick fact api

### PR DESCRIPTION
**Pull request by odinr**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "@equinor/fusion-components",
-    "version": "1.3.13",
+    "version": "1.3.13-pr792.14",
```
Comment (eskil0312) [inline package.json:3]: This is for testing right? Should be removed before merge

File: src/components/general/ApplicationGuidance/components/Anchor.tsx [hunk 2]
```diff
@@ -1,54 +1,85 @@
...
 
-export type ApplicationGuidanceAnchorProps = React.PropsWithChildren<AppGuideAnchorElementProps>;
+export interface AppGuideAnchorRef<R extends HTMLElement> {
+    id: string;
+    scope: string;
+    padding?: number;
+    ref: React.RefObject<R>;
+}
+
+export const useAnchor = <R extends HTMLElement>(anchor: Omit<AppGuideAnchorRef<R>, 'ref'>) => {
```
Comment (eskil0312) [inline src/components/general/ApplicationGuidance/components/Anchor.tsx:17]: Maybe add some documentation/comments for this. The hook name "useAnchor" is not very self explanatory

File: src/customElements/components/application-guide/element.ts [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+    protected _handleQuickFactShow(e: QuickFactEvent<QuickFactEventType.show>) {
+        const { anchor, scope, info } = e.detail;
+        const detail: ApplicationGuideEventDetail = {
+            selected: { anchor, scope },
+            info
+        };
+        this._dispatchEvent(ApplicationGuideEventType.show, { detail });
+    }
+
+    // @TODO
```
Comment (eskil0312) [inline src/customElements/components/application-guide/element.ts:116]: todo what?

File: src/customElements/components/base/api.ts [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import { IFusionContext, ApiClients } from '@equinor/fusion';
+
+const globalEquinorFusionContextKey = '74b1613f-f22a-451b-a5c3-1c9391e91e68';
+
+declare global {
+    interface Window { clientId: string; }
+}
+
+/** @TODO */
```
Comment (eskil0312) [inline src/customElements/components/base/api.ts:9]: todo what?

File: package.json (hunk 1 above)
Comment (odinr) [reply package.json:3]: this will change before when versjoning

**Review by eskil0312 — APPROVED** [review]

**Pull request merged**